src/patternfly/components/Divider/divider.scss

```diff
@@ -1,9 +1,63 @@
+$pf-c-divider--breakpoint-map: build-breakpoint-map("base", "sm", "md", "lg", "xl", "2xl");
+$pf-c-divider--spacer-map: build-spacer-map("none", "sm", "md", "lg", "xl", "2xl", "3xl");
+
 .pf-c-divider {
+  // update during breaking change
   --pf-c-divider--Height: var(--pf-global--BorderWidth--sm);
   --pf-c-divider--BackgroundColor: var(--pf-global--BorderColor--100);
+  --pf-c-divider--after--Height: var(--pf-c-divider--Height);
+  --pf-c-divider--after--BackgroundColor: var(--pf-c-divider--BackgroundColor);
+
+  // end - update during breaking change
+
+  --pf-c-divider--m-vertical--Width: var(--pf-global--BorderWidth--sm);
 
-  height: var(--pf-c-divider--Height);
-  background-color: var(--pf-c-divider--BackgroundColor);
+  // After
+  --pf-c-divider--after--FlexBasis: 100%;
+  --pf-c-divider--after--Inset: 0%;
+
+  display: flex;
+  align-items: center;
+  align-self: stretch;
+  justify-content: center;
+  width: 100%;
   border: 0; // removes the default border styling on an hr
-  flex-shrink: 0;
+
+  &::after {
+    flex-basis: calc(var(--pf-c-divider--after--FlexBasis) - calc(var(--pf-c-divider--after--Inset) * 2));
```

Comment:
Looks like any time `--pf-c-divider--after--Inset` is `0`, the `calc()` is going to fail due to a type mismatch.

Reply:
Good catch!